frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -9,6 +9,7 @@ import { VMTemplateLink } from '../vm-templates/vm-template-link';
 import { getBasicID, prefixedID } from '../../utils';
 import { vmDescriptionModal, vmFlavorModal } from '../modals';
 import { VMCDRomModal } from '../modals/cdrom-vm-modal';
+import { VMBootOrderModal } from '../modals/vm-boot-order-modal';
```

Comment:
yes :+1: , before that it will need some twiking with the k8s save function, so I can't use it their yet.

will add it to the items to do in the description above.

Reply:
have you tried `getVMLikePatches()` ? or is there some other problem?

Reply:
hmmm :-) didn't know about that ... looking ...

Reply:
using the `getVMLikePatches` method :+1:

should work for templates now too, didn't check  :-)

Reply:
done, works for templates as expected :-)